website/blog/2021-08-26-many-platform-vision.md

```diff
@@ -47,7 +47,7 @@ As another example, our research into user perception of speed on the web led to
 
 In addition to domain-specific engineers and meetups and conferences, each platform also brings other unique players solving similar problems. On the web, React (which directly powers React Native) frequently draws inspiration from other open source web frameworks like [Vue](https://vuejs.org/), [Preact](https://preactjs.com/), and [Svelte](https://svelte.dev/). On mobile, React Native has been inspired by other open source mobile frameworks, and we have been learning from other mobile frameworks built inside Facebook.
 
-<!-- alex ignore challenged -->
```

Comment:
Where does alex define these mappings i.e (special -> "challenged")?

Reply:
Not sure TBH 😅 I have used the error messages to correct that. 

They include the correct keyword at the end of the error messages.